webapp/channels/src/components/widgets/separator/notification-separator.tsx

```diff
@@ -1,26 +1,15 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
 import React from 'react';
```

Comment:
We prefer to only migrate 1 component per PR, please only include here the one that is related to the `email_icon.tsx`migration.